packages/mobile/src/app/saga.test.ts

```diff
@@ -90,6 +96,18 @@ describe('App saga', () => {
       .run()
     expect(navigate).toHaveBeenCalledWith(testRoute.routeName, testRoute.params)
   })
+
+  it('Handle Dappkit deep link', async () => {
```

Comment:
Nit note to ourself: looks like we're inconsistent with naming these in this file compared to the others.
I've seen a lot of "verb xxx" elsewhere: should be "handles Dappkit deep line".